testsuites/syncgateway/functional/tests/test_log_rotation.py

```diff
@@ -26,9 +27,15 @@ def load_sync_gateway_config(sync_gateway_config, mode, server_url, xattrs_enabl
         else:
             autoimport_prop = ""
             xattrs_prop = ""
+        couchbase_server_primary_node = ""
+        cbs_servers = get_cbs_servers(cluster_config)
+        for i in range(len(cbs_servers)):
+            couchbase_server_primary_node = couchbase_server_primary_node + cbs_servers[i]
+            if(i + 1 < len(cbs_servers)):
+                couchbase_server_primary_node = couchbase_server_primary_node + ","
```

Comment:
This code with a comma separated list of servers is repeated everywhere. It would be a good idea to separate this out as a function in utils.py

Reply:
Added to utils.py and calling from there. Updated all python script which is having these steps